src/services/infrastructure/url-generator/url.generator.service.ts

```
@@ -81,24 +81,20 @@ export class UrlGeneratorService {
         const userEntityInfo = await this.getNameableEntityInfoForProfileOrFail(
           'user',
           profile.id
         );
         return this.createUrlForUserNameID(userEntityInfo.entityNameID);
-      }
-      case ProfileType.VIRTUAL_CONTRIBUTOR: {
+      case ProfileType.VIRTUAL_CONTRIBUTOR:
         const vcEntityInfo = await this.getNameableEntityInfoForProfileOrFail(
           'virtual_contributor',
           profile.id
         );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)